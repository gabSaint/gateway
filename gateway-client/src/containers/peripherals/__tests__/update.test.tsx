import React from "react";
import { ReactWrapper, shallow, ShallowWrapper } from "enzyme";

describe("Updateeripheral", () => {
  it("", () => {
    expect(1).toBe(1);
  });
});
