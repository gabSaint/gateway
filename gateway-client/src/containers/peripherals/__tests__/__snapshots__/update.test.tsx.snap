// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpdatePeripheral renders correctly 1`] = `
<div
  className="contain-row"
>
  <h2>
    Edit Peripheral
  </h2>
  <button
    className="negative"
    data-test="form-button-delete"
    onClick={[Function]}
  >
    Delete
  </button>
</div>
`;
